crates/pallet-evm-precompiles-constant-accounts/src/lib.rs

```diff
@@ -0,0 +1,51 @@
+//! A substrate pallet.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use frame_support::traits::StorageVersion;
+pub use pallet::*;
+use sp_core::H160;
+
+#[cfg(test)]
+mod mock;
+
+#[cfg(test)]
+mod tests;
+
+/// The current storage version.
+const STORAGE_VERSION: StorageVersion = StorageVersion::new(0);
+
+// We have to temporarily allow some clippy lints. Later on we'll send patches to substrate to
+// fix them at their end.
+#[allow(clippy::missing_docs_in_private_items)]
+#[frame_support::pallet]
+pub mod pallet {
+    use frame_support::pallet_prelude::*;
+
+    use super::*;
+
+    #[pallet::pallet]
+    #[pallet::storage_version(STORAGE_VERSION)]
+    #[pallet::generate_store(pub(super) trait Store)]
+    pub struct Pallet<T>(_);
+
+    /// Configuration trait of this pallet.
+    #[pallet::config]
+    pub trait Config: frame_system::Config + pallet_evm::Config {}
+
+    #[pallet::genesis_config]
+    #[derive(Default)]
+    pub struct GenesisConfig {
+        /// The list of precompiles to be stored as account codes at evm.
+        pub precompiles: Vec<(H160, Vec<u8>)>,
+    }
```

Comment:
```suggestion
    pub struct GenesisConfig {
    }
```

We should instead configure the set of accounts via the `Config` (via a `Get<Vec<H160>>`).

There also should be just a built-in code in this pallet uses, no need to make it configurable.